.github/workflows/openbsd.yml

```diff
@@ -0,0 +1,48 @@
+name: C CI on OpenBSD
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: OpenBSD-${{ matrix.config.name }}
+    runs-on: macos-12
+    env:
+      CFLAGS: ${{ matrix.config.cflags }}
+      NROFF: ${{ matrix.config.nroff }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+         - {
+           name: "default64",
+           nroff: "nroff",
```

Comment:
In case this fails, maybe `mandoc` would work better then.

Reply:
At the moment it just works.